src/Libraries/CoreNodes/ProtoGeometryHelper.cs

```diff
@@ -0,0 +1,15 @@
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using System;
+
+namespace DSCore
+{
+    [IsVisibleInDynamoLibrary(false)]
+    public class BoundaryConditionHelper
+    {
+        public static PanelSurfaceBoundaryCondition BoundaryConditionFromString(string val)
+        {
+            return Enum.Parse<PanelSurfaceBoundaryCondition>(val);
+        }
+    }
+}
```

Comment:
I did not figure out a better way to expose the PanelSurfaceBoundaryCondition type as the new nodes output.
@aparajit-pratap any other ideas besides a vm exposed function ?

Reply:
Hmm, that's probably because it's an enum and not a class. I wonder if there's a better way myself 🤔

Reply:
I guess one way to improve this in general would be to add this to add enum support to https://github.com/DynamoDS/Dynamo/blob/0b23a2e98f9d62945ee2d44042fda8997cd3a775/src/Engine/ProtoCore/Parser/AssociativeAST.cs#L2781

Reply:
Looks a little more complicated. The way it is written now, the enum parse function needs to be exposed to the vm.
The Protocore project does not have the proper dependencies to make this easy.
Since it seems like a bigger job, I would leave it for another task.